Dockerfile

```diff
@@ -0,0 +1,31 @@
+# Use the custom base image with Ruby, Node.js, npm, and Shopify CLI pre-installed
+FROM aaronmedinaaligent/shopify-cli-base:latest
```

Comment:
We should switch this out to either a public image or build our own in the Aligent namespace (public)

Reply:
Thanks Gowri for the feedback, that is the plan. It is just temporary to make things work. I will be coordinating with Chris to set up the docker image. :)